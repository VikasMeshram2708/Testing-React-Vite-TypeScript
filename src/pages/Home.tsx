export default function Home() {
  return (
    <section data-testId="homePage">
      <h1>Home Page</h1>
    </section>
  );
}
