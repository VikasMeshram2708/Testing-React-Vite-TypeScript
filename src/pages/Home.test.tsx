import { render, screen } from "@testing-library/react";
import Home from "./Home";

describe("renders learn react link", () => {
  it("testing heading", () => {
    render(<Home />);
    const HomePage = screen.getByTestId("homePage");
    expect(HomePage).toMatchSnapshot();
  });
});
