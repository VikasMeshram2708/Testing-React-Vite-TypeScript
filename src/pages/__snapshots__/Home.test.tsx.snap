// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link testing heading 1`] = `
<section
  data-testid="homePage"
>
  <h1>
    Home Page
  </h1>
</section>
`;
